ramls/request-by-instance-id.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "A request for an item based on the specified instance ID",
+  "description": "Request for an item selected from the that might be at a different location or already checked out to another patron",
+  "type": "object",
+  "properties": {
```

Comment:
Do we need all of these properties for an instance level request?

For example, can they be made by proxy? Will a client / user be specifying both of the expiry dates? Can the different fulfilment preferences be chosen?

Reply:
That's a good question. I didn't have specific requirements for the properties, so I kept them as close to the item level requests as possible. From the point of view of EDS, I could strip out several properties, like `proxyUserId`, `holdShelfExpriationDate`, `deliveryAddressTypeId`, and fulfilmentPreference` (always set to `Hold Shelf` on request creation).

Would you rather we started with a smaller set of properties? I am fine with making the schema simpler and growing it as needed.

Reply:
I'd rather grow it as needed (it is easier not to add something than to remove it), especially if most of these properties are likely to be optional.

If there are known upcoming required properties that is different.